requirements.txt

```diff
@@ -1,15 +1,17 @@
-numpy
-pandas
-scikit-learn
-transformers
-torch
-sentencepiece
-pydantic==1.10.6
-spacy
-johnsnowlabs==4.3.5
-plotly
-asyncio
-nest-asyncio
-jsonlines
-openai
-langchain
\ No newline at end of file
+numpy
+pandas
+scikit-learn
+transformers
+torch
+sentencepiece
+pydantic==1.10.6
+spacy
+johnsnowlabs==4.3.5
+plotly
+asyncio
+nest-asyncio
+jsonlines
+openai
+langchain
+spacy

```

Comment:
we dont need spacy and en_core_web_sm here, they are optional to install

Reply:
I have made the other two changes as requested and updated the Class name in the robustness.py file and in test_robustness.py where the class name is being called. The original code with which the issue was associated used spacy and en_core_web_sm as it requires in the further loop so what should I do about that.